src/containers/PDiskPage/PDiskPage.tsx

```diff
@@ -118,7 +119,7 @@ export function PDiskPage() {
                 <ButtonWithConfirmDialog
                     onConfirmAction={handleRestart}
                     onConfirmActionSuccess={handleAfterRestart}
-                    buttonDisabled={!nodeId || !pDiskId}
+                    buttonDisabled={!nodeId || !pDiskId || !isUserAllowedToMakeChanges}
```

Comment:
Here and farther I think we should add a tooltip with information why is this button disabled.

Reply:
Added